Source/SvgAttributeCollection.cs

```diff
@@ -39,32 +39,48 @@ public TAttributeType GetAttribute<TAttributeType>(string attributeName, TAttrib
         /// </summary>
         /// <typeparam name="TAttributeType">The type of the attribute value.</typeparam>
         /// <param name="attributeName">A <see cref="string"/> containing the name of the attribute.</param>
+        /// <param name="inherited">The attribute is inherited or not.</param>

```

Comment:
I would like a better description here. As I understand, `inherited` is only used if the value hasn't been specified yet, to differentiate between using the inherited value, if any, and the default value.